### ISSUE TITLE
Cannot delete single entry

### ISSUE THREAD

**Issue by Giois93** [sean-smith/chromeshortcuts#1]
Simply, I cannot delete a single entry once added to the list. 
I can only clear them all.

**Comment by sean-smith** [comment]
Yep, I built it without this feature. Was planning to add it but got busy. Feel free to submit at PR. The code is actually already there to do this. I have a function `remove()` in `popup.js` just need to create a ui for it. If I recall, I didn't do it because I was getting weird wrapping with the svg that I was using.

**Comment by sean-smith** [comment]
I was thinking on the UI, this https://www.iconfinder.com/icons/2561505/x_icon#size=24 "X" to the right of the "Submit" button

**Comment by sean-smith** [comment]
Fixed this, will be out in the Chrome store soon. Until then you can get the feature by loading in the extension manually, via instructions [here](https://github.com/sean-smith/chromeshortcuts#contributing)
